test/test_redcarpet.rb

```diff
@@ -46,16 +46,17 @@ class TestRedcarpet < JekyllUnitTest
       end
 
       should "render fenced code blocks with syntax highlighting" do
-        assert_equal "<div class=\"highlight\"><pre><code class=\"language-ruby\" "\
-                     "data-lang=\"ruby\"><span class=\"nb\">puts</span> <span "\
-                     "class=\"s2\">&quot;Hello world&quot;</span>\n</code></pre></div>",
-                     @markdown.convert(
-                       <<-EOS
+        assert_equal \
+          "<div class=\"highlight\"><pre><code class=\"language-ruby\" " \
+          "data-lang=\"ruby\"><span></span><span class=\"nb\">puts</span> <span " \
+          "class=\"s2\">&quot;Hello world&quot;</span>\n</code></pre></div>",
```

Comment:
We can ideally avoid escaping the double-quote here by using `%{}` like in `test_tags.rb`..
But on further inspection, there are multiple such instances.. best handled in a separate PR.. :+1:

Reply:
Feel free to take over this PR and commit directly to this branch.